src/main/java/core/basesyntax/service/impl/FruitTransactionServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.dao.FruitTransactionDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionService;
+
+public class FruitTransactionServiceImpl implements FruitTransactionService {
+    private final FruitTransactionDao fruitTransactionDao;
+
+    public FruitTransactionServiceImpl() {
+        fruitTransactionDao = new FruitTransactionDaoImpl();
+    }
+
+    @Override
+    public void createFruitTransaction(String operationCode, String fruit, int quantity) {
```

Comment:
You can return fruitTransaction from this method to loop in parseData() for creating a list of FruitTrnsactions